tools/src/main/java/org/apache/kafka/trogdor/coordinator/TaskManager.java

```diff
@@ -288,25 +291,103 @@ private JsonNode getCombinedStatus() {
     }
 
     /**
-     * Create a task.
-     *
-     * @param id                    The ID of the task to create.
-     * @param spec                  The specification of the task to create.
+     * Creates and schedules a task.
      */
-    public void createTask(final String id, TaskSpec spec)
+    public void createAndScheduleTask(TaskDetail task)
```

Comment:
Like I wrote above, we should just have one function that takes a map.  Then we don't need a special function for the case where there's only one task to create.